packages/yourloops/pages/hcp/teams-list.tsx

```diff
@@ -107,6 +127,12 @@ class TeamsListPage extends React.Component<RouteComponentProps, TeamsListPageSt
             {teamsItems}
           </Grid>
         </Container>
+        <RemoveMemberDialog userToBeRemoved={userToBeRemoved} handleClose={this.onHideModalRemoveMember} handleRemoveTeamMember={this.onRemoveTeamMember} />
+        <Snackbar open={apiReturnAlert !== null} autoHideDuration={6000} onClose={this.onCloseAlert} anchorOrigin={{ vertical: 'top', horizontal: 'center' }}>
+          <Alert onClose={this.onCloseAlert} severity={apiReturnAlert?.severity}>
+            {apiReturnAlert?.message}
+          </Alert>
```

Comment:
Looks like we are using the default Alert but the snackBar is not visible with the light green for success on top of the green navigation bar.
Something that may have to be reviewed globally for snackBar

Reply:
OK